### PR TITLE
Add Student

### PR DESCRIPTION
**Pull request by hieungo89**
Added email field for when adding a new student and added the route to return to student list when in individual student's profile.

### CODE REVIEW

File: src/backend/db/zapatos/.eslintrc.json
Comment (amantri) [inline src/backend/db/zapatos/.eslintrc.json:1]: Undo the changes done by prettier in this file. This file is not ignored by prettier (after you sync to head)

File: src/backend/routers/students.ts [hunk 1]
```diff
@@ -25,14 +25,20 @@ export const studentProcedures = {
...
+        first_name: z.string(),
+        last_name: z.string(),
+        email: z.string(),
+      })
+    )
     .mutation(async (req) => {
-      const { first_name, last_name } = req.input;
+      const { first_name, last_name, email } = req.input;
 
       // todo: add a unique constraint to prevent duplicate students
```
Comment (amantri) [inline src/backend/routers/students.ts:38]: This todo can now be removed

File: src/backend/routers/students.test.ts [hunk 2]
```diff
@@ -25,6 +26,7 @@ test("getAllStudents", async (t) => {
     .values({
       first_name: "Foo",
       last_name: "Bar",
+      email: "foo.bar@email.com",
```
Comment (amantri) [inline src/backend/routers/students.test.ts:29]: It would be good to add a test case that tries to insert a duplicate email into the db.

File: src/backend/db/zapatos/.eslintrc.json
Comment (hieungo89) [reply src/backend/db/zapatos/.eslintrc.json:1]: Didn't realize that this has been changed. Will revert it.

File: src/backend/routers/students.test.ts (hunk 2 above)
Comment (hieungo89) [reply src/backend/routers/students.test.ts:29]: I added the new test case for duplicate emails. Please take a look and also let me know if the naming is accurate to describe the test.

**Review by amantri — APPROVED** [review]
